MediaGalleryUi/view/adminhtml/web/js/grid/columns/image/actions.js

```diff
@@ -123,86 +86,12 @@ define([
         },
 
         /**
-         * Delete image
-         *
-         * @param {Object} record
-         */
-        deleteImage: function (record) {
-            var recordId = this.imageModel().getId(record);
-
-            $.ajax({
-                type: 'POST',
-                url: this.imageModel().deleteImageUrl,
-                dataType: 'json',
-                showLoader: true,
-                data: {
-                    'image_id': recordId
-                },
-                context: this,
-
-                /**
-                 * Success handler for deleting image
-                 *
-                 * @param {Object} response
-                 */
-                success: function (response) {
-                    var message = !_.isUndefined(response.message) ? response.message : null;
-
-                    if (!response.success) {
-                        message = message || $.mage.__('There was an error on attempt to delete the image.');
-                        this.addMessage('error', message);
-
-                        return;
-                    }
-
-                    message = message || $.mage.__('You have successfully removed the image.');
-                    this.reloadGrid();
-                    this.addMessage('success', message);
-                    $(this.imageModel().deleteSelectedBtnSelector).addClass('no-display');
-                    $(this.imageModel().addSelectedBtnSelector).addClass('no-display');
-                }.bind(this),
-
-                /**
-                 * Error handler for deleting image
-                 *
-                 * @param {Object} response
-                 */
-                error: function (response) {
-                    var message;
-
-                    if (typeof response.responseJSON === 'undefined' ||
-                        typeof response.responseJSON.message === 'undefined'
-                    ) {
-                        message = 'There was an error on attempt to delete the image.';
-                    } else {
-                        message = response.responseJSON.message;
-                    }
-
-                    this.addMessage('error', message);
-                }.bind(this)
-            });
-        },
-
-        /**
-         * Reload grid
-         */
-        reloadGrid: function () {
-            var provider = this.provider(),
-                dataStorage = provider.storage();
-
-            dataStorage.clearRequests();
-            provider.reload();
-        },
-
-        /**
-         * Add message
+         * Get target element
          *
-         * @param {String} code
-         * @param {String} message
+         * @returns {*|n.fn.init|jQuery|HTMLElement}
          */
-        addMessage: function (code, message) {
-            this.messages().add(code, message);
-            this.messages().scheduleCleanup();
+        getTargetElement: function () {
```

Comment:
Why we need this method ? i can't find where it used

Reply:
I have removed this method. Thanks